operators/log2rbac/0.0.5/manifests/log2rbac-operator.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,184 @@
+# Generated by https://github.com/upbound/olm-bundle
+apiVersion: v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"kremser.dev/v1","kind":"RbacNegotiation","metadata":{"name":"for-prometheus-operator"},"spec":{"for":{"kind":"Deployment","name":"prometheus-operator","namespace":"monitoring"},"role":{"createIfNotExist":true,"isClusterRole":true,"name":"prometheus-operator-role"}}}]'
+    capabilities: Seamless Upgrades
+    categories: Developer Tools, Integration & Delivery
+    certified: "false"
+    containerImage: jkremser/log2rbac:v0.0.5
```

Comment:
Please add registry host to image. Probably `docker.io/jkremser/log2rbac:v0.0.5`